screener.config.js

```diff
@@ -28,15 +30,28 @@ const screenerConfig = {
   // screenshot every example in maximized mode
   states: glob
     .sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] })
-    .map(examplePath => {
-      const { name: nameWithoutExtension, base: nameWithExtension } = path.parse(examplePath)
+    .reduce((states, examplePath) => {
+      const { name: nameWithoutExtension, base: nameWithExtension, dir } = path.parse(examplePath)
       const rtl = nameWithExtension.endsWith('.rtl.tsx')
+      const url = `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}/${rtl}`
 
-      return {
-        url: `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}/${rtl}`,
+      states.push({
+        url,
         name: nameWithExtension,
-      }
-    }),
+        steps: fs.existsSync(`${dir}/${nameWithoutExtension}.steps.js`)
```

Comment:
please, introduce reference to the corresponding Screener docs file in the comment